lib/extras/dec/apng.cc

```diff
@@ -257,6 +257,17 @@ Status DecodeChrmChunk(Bytes payload, JxlColorEncoding* color_encoding) {
   return true;
 }
 
+/** Extract information from 'cLLi' chunk. */
```

Comment:
```suggestion
/** Extracts information from 'cLLi' chunk. */
```

(https://google.github.io/styleguide/cppguide.html#Function_Comments

> Function comments should be written with an implied subject of _This function_ and should start with the verb phrase; for example, "Opens the file", rather than "Open the file".

)